app/components/Settings/Settings.react.js

```diff
@@ -440,6 +462,12 @@ class Settings extends Component {
                                                 FAQ while you wait! 📰`}
                                             </p>
                                         </div>
+                                    ): (
+                                        <div>
+                                            <p>
+                                                <a onClick={() => window.location.assign('/login')}>Log into Plotly</a>
```

Comment:
The message should be a little more informing imho:

> Log in to Plotly to generate unique SSL certificates, schedule and save queries.

@chriddyp thoughts?

Reply:
yeah, that sounds good to me